merino-integration-tests/src/suggest.rs

```diff
@@ -1,6 +1,9 @@
 //! Tests Merino's ability to make basic suggestions.
-#![cfg(test)]
+// Note, setting the following line will cause a compile error. This may be due to the
+// fact that the proc_macro may not be able to locate items.
+//#![cfg(test)]
```

Comment:
Instead of removing this, we should move `setup_remote_settings_collection` into the testing tools file, since it is only used there now.

Reply:
Ah, far better suggestion. thanks!